src/Proto/nodeguard.proto

```diff
@@ -57,7 +57,12 @@ service NodeGuardService {
   /*
   Gets a list of available UTXOs for a wallet
    */
-  rpc GetAvailableUtxos(GetAvailableUtxosRequest) returns (GetAvailableUtxosResponse);
+  rpc GetAvailableUtxos(GetAvailableUtxosRequest) returns (GetUtxosResponse);
+
+  /*
+  Gets a list of UTXOs
```

Comment:
If this is all the UTXOs in the treasury, please doc it

Reply:
documented better